core/GlobalState.cc

```diff
@@ -2238,6 +2250,7 @@ unique_ptr<LocalSymbolTableHashes> GlobalState::hash() const {
     uint32_t methodHash = 0;
     UnorderedMap<WithoutUniqueNameHash, uint32_t> methodHashesMap;
     UnorderedMap<WithoutUniqueNameHash, uint32_t> staticFieldHashesMap;
+    UnorderedMap<WithoutUniqueNameHash, uint32_t> fieldHashesMap;
```

Comment:
Do we actually need to keep these as separate? Would it be possible to rename `methodHashesMap` to simply `deleteableSymbolHashesMap` (or some better name)?

I think it would mean that apart from some slight renames in LSPFileUpdates.cc, the core logic there wouldn't have to change.

And then when non-class alias static fields also take the fast path, they would also be able to use the same logic.

Reply:
I don't have a strong opinion about this.  I have mild FUD about throwing everything into one big bin and resulting collisions vs. multiple small bins and fewer/no collisions thanks to the binning...but I don't know that it actually matters, and it would be nice to not have to update a zillion places with almost the exact same code for different symbol types.

It is slightly more consistent with the strong typing we have for symbols to have separate bins for things, I think...but if we treat those bins effectively the same, does the distinction really matter? :thinking: